ratelimit.go

```diff
@@ -53,8 +54,7 @@ type state struct {
 }
 
 type limiter struct {
-	state   unsafe.Pointer
-	padding [56]byte // cache line size - state pointer size = 64 - 8; created to avoid false sharing
```

Comment:
You can't just remove padding, it is crucial to maintain performance properties of this rate limiter in case of collocation with other frequently accessed memory.

Reply:
Fair enough - I admit I didn't read/think this is important.

Added an explicit linter exception instead + copied over your comment.